### PR TITLE
Slam 70 - Add ramp to IGVC world

### LINKED ISSUE #70: Add ramp to IGVC world
**Description**:
- In the actual track of the competition there are ramps that the robots need to go over. These ramps need to be added to the simulation world

**Goals**:
- [x] A ramp similar in size and angle to the igvc ramp is added to a gazebo simulation world

**Acceptance Criteria**:
- [x] Ramp exists when launching IGVC world in Gazebo

### PR DESCRIPTION
**Pull request by 1999michael**
Did not test by running `roslaunch caffeine caffeine_gazebo.launch world:=absolute_path` as didn't want to focus too much time to debug why that way wouldn't work for me. Instead, to test ramp in IGVC world, run `gazebo ~/SLAM/slam-ws/src/igvc_world/worlds/igvc_basic.world`.

`igvc_basic.world` has the edits with some obstacles removed to place ramps in approximately correct place.
`original_igvc` has the unaltered file.

Color is not set correct, nor are white lines on ramp

### CODE REVIEW

**Review by chandra-gummaluru — APPROVED** [review]
Looks good!

A question for thought: What are the grades of these ramps? Should we make them different? We might want to have at least one of them be 15% (the maximum possible from the rules).

**Comment by 1999michael** [comment]
> Looks good!
> 
> A question for thought: What are the grades of these ramps? Should we make them different? We might want to have at least one of them be 15% (the maximum possible from the rules).

I set the ramps all to 15% incline

File: slam-ws/src/nav_rtab/config/rtab_config.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 subscibe_depth: true
-subscribe_laserScan: true
+subscribe_scan: true
```
Comment (erickmu1) [inline slam-ws/src/nav_rtab/config/rtab_config.yaml:2]: This was because `laserScan` is deprecated correct?

**Review by erickmu1 — APPROVED** [review]
@1999michael could you make a new issue that concern adding textures to the ramps

**Pull request merged**